### PR TITLE
gop32 configuration support in xeve main profile

### PR DESCRIPTION
**Pull request by mamthan**

### CODE REVIEW

File: src_base/xeve_def.h [hunk 1]
```diff
@@ -822,7 +822,8 @@ typedef struct _XEVE_SBAC_CTX
 /* rpl structure */
 #define XEVE_MAX_NUM_REF_PICS                   21
 #define XEVE_MAX_NUM_ACTIVE_REF_FRAME           5
-#define XEVE_MAX_NUM_RPLS                       32
+
+#define XEVE_MAX_NUM_RPLS                       64
```
Comment (dariusz-f) [inline src_base/xeve_def.h:826]: Duplicated definition. In heve.h should be enough, so you can remove define from this file.

File: inc/xeve.h [hunk 2]
```diff
@@ -491,9 +491,9 @@ typedef struct _XEVE_PARAM
     int            arbitrary_slice_flag;
     char           num_remaining_tiles_in_slice_minus1[XEVE_MAX_NUM_TILES >> 1];
     int            rpl_extern;
-    /* max num of RPL is 32 */
-    char           rpl0[32][256];
-    char           rpl1[32][256];
+    /* max num of RPL is 64 after gop32 support */
```
Comment (dariusz-f) [inline inc/xeve.h:494]: Remove "after gop32 support" this info is superfluous.

**Review by dariusz-f — CHANGES_REQUESTED** [review]

**Review by dariusz-f — APPROVED** [review]
looks good

**Review by mss-park — APPROVED** [review]
Looks good to me.

**Pull request merged**